yaptide/routes/utils/tokens.py

```diff
@@ -1,14 +1,14 @@
 from datetime import datetime, timedelta
 from secrets import token_hex
 from typing import Union
-
 import jwt
 
 SECRET_KEY_TOKEN = token_hex(256)
 SECRET_KEY_TOKEN_REFRESH = token_hex(256)
 _Refresh_Token_Expiration_Time = 120  # minutes
 _Access_Token_Expiration_Time = 10  # minutes
 _Keycloak_Token_Expiration_Time = 30  # minutes
+_Simulation_Token_Expiration_time = 7  # days
```

Comment:
why 7 days ?

Reply:
not my decision, just added extra space here.